include/dump_offload.hpp

```diff
@@ -0,0 +1,518 @@
+#pragma once
+
+#include <sys/select.h>
+
+#include <boost/asio.hpp>
+#include <boost/asio/basic_socket_acceptor.hpp>
+#include <boost/asio/basic_stream_socket.hpp>
+#include <boost/asio/local/stream_protocol.hpp>
+#include <boost/beast/core/flat_static_buffer.hpp>
+#include <boost/beast/http.hpp>
+#include <http_stream.hpp>
+#include <ibm/utils.hpp>
+
+#include <random>
+
+namespace crow
+{
+namespace obmc_dump
+{
+
+std::string unixSocketPathDir = "/tmp/DumpOffloadSockets/";
+
+inline void handleDumpOffloadUrl(const crow::Request& req, crow::Response& res,
+                                 const std::string& entryId,
```

Comment:
```
#include <string>
```

Reply:
is there any change in string usage in coding guidelines?

Reply:
You should include what you use

Reply:
done